build.gradle.kts

```diff
@@ -274,16 +306,17 @@ tasks {
     from(sourceSets.main.get().java) {
       include("**/com/maddyhome/idea/vim/**/*.java")
     }
-    from(project(":vim-engine").sourceSets.main.get().java) {
-      include("**/com/maddyhome/idea/vim/**/*.java")
-    }
+    // TODO: This causes errors with sourceSets not be available for this project?
+//    from(project(":vim-engine").sourceSets.main.get().java) {
+//      include("**/com/maddyhome/idea/vim/**/*.java")
+//    }
```

Comment:
HELP! I have no idea why the build breaks when this is uncommented. The `vim-engine` module hasn't been modified at all, and this used to work with the old gradle plugin. I don't know what could be wrong.

Reply:
This `project(":vim-engine").sourceSets.main.get().java` is basically wrong approach. I'll take a look at this soon.

Reply:
Properly implemented. Thank Jakub!